roles/confluent.ksql/defaults/main.yml

```diff
@@ -50,3 +50,5 @@ ksql:
     ksql.streams.producer.max.block.ms: 9223372036854775807
     producer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
     consumer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+    ksql.logging.processing.topic.auto.create: "{{ksql_log_streaming_enabled}}"
+    ksql.logging.processing.topic.name: "{{ksql_service_id}}{{ksql_processing_log}}"
```

Comment:
Should these be moved into the properties template with an if statement around them?
Also I think you'll need the `lower` filter:
```
ksql.logging.processing.topic.auto.create={{ksql_log_streaming_enabled|lower}}
```

Reply:
We could but it doesn't really impact anything as nothing is done unless the feature is enabled.

Reply:
I'll move it.